include/seqan/index/index_qgram.h

```diff
@@ -1404,6 +1404,7 @@ typename TBucketMap,
 typename TText,
 typename TShape,
 typename TStepSize >
+/*[[deprecated("Use createQGramIndex(index) instead.")]]*/
```

Comment:
The function ```createQGramIndex(sa, dir , ...)``` is deprecated but used by the app searchjoin. It requires some work in my POV to change the usage so thought leave the comment such that in seqan3 where we might change or remove the application anyway we can include the deprecation macro then.
Shall I remove the comment?